stacks/api/function/src/function.controller.ts

```diff
@@ -102,7 +102,7 @@ export class FunctionController {
 
     return triggers
       .filter(trigger => trigger.type == "bucket")
-      .some((trigger, index) => {
+      .some((trigger, index, triggers) => {
         const foundIndex = triggers.findIndex(
```

Comment:
`triggers.findIndex` checks each trigger of triggers array which is not filtered for only bucket triggers and it causes to find the wrong index. It must check only filtered functions to find the correct indexes.